test/data/brave-wallet/cardano/cardano_sdk_crypto_slip23/index.mts

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Bip32Ed25519, SodiumBip32Ed25519 } from "@cardano-sdk/crypto";
+import { HexBlob } from "@cardano-sdk/util";

```

Comment:
Shouldn't we need to add node deps for these?
Also I don't quite understand what this file is for, we don't export any providers in this PR. How do we test the test vectors against our implementation through this file?

Reply:
This file is just a recipe of how `test_vectors.json` was generated and is not included in CI. Keeping it close to `test_vectors.json` in case we want to extended these tests or change it in any way.

Reply:
Oh, I misunderstood the question. 
The test is `HDKeyEd25519Slip23UnitTest.CardanoSdkCryptoSlip23`